code/modules/html_interface/map/station_map.dm

```diff
@@ -158,11 +160,13 @@
 									continue
 								else if ((istype(tile, /turf/unsimulated/floor/snow/permafrost) && istype(aera, /area/surface/mine)) ||(istype(tile, /turf/unsimulated/floor/snow/cave) && istype(aera, /area/surface/outer/ne)))
 									override = TRUE
-								else if (istype(aera, /area/mine/explored) && !istype(tile, /turf/unsimulated/floor/airless))
-									if (prob(80)) //blurring the shape of Snaxi's Kraken asteroid so it's a bit more subtle.
-										override = TRUE
+
 								else if (istype(aera, /area/derelict/secret))
 									override = TRUE
+							if (istype(Z, /datum/zLevel/krakenroid))
```

Comment:
instead of a typecheck, a var on the zlevel would be more flexible; you could make this binary or you could even make it a list of areas in which to do the blurring

imagine if you will lamprey could set some areas to blur on the main station through use of this var

Reply:
That's a good point, here we go.